### DISCUSSION TITLE
Lovelace YAML

### DISCUSSION THREAD

**Discussion by onyxhat** [G-Two/homeassistant-subaru#13] [Q&A]
Could you share the lovelace yaml for the dashboard shown in the picture in this repo? I think it would provide a great starting point for many new users (myself included).

**Comment by chvvkumar** [comment]
> Could you share the lovelace yaml for the dashboard shown in the picture in this repo? I think it would provide a great starting point for many new users (myself included).

I shared my dashboard here using the dashboard @G-Two created:

https://github.com/chvvkumar/subaru_outback

**Reply by onyxhat** [reply]
Thank you!

**Reply by BovBrew** [reply]
@chvvkumar thanks for the help! This got me started.
